src/spyglass/spikesorting/v1/metric_curation.py

```diff
@@ -239,9 +239,14 @@ def make(self, key):
         os.makedirs(waveforms_dir, exist_ok=True)
 
         logger.info("Extracting waveforms...")
+        if "sparse" in waveform_params.keys():
```

Comment:
I think this could just be:
`sparse = waveform_params.get("sparse", False)`

You could put it directly in the call to `extract_waveforms`

Reply:
Also please fix linting and update the change log.